zerver/lib/notifications.py

```diff
@@ -307,8 +307,9 @@ def do_send_missedmessage_events(user_profile, missed_messages, message_count):
         # If we have one huddle, set a reply-to to all of the members
         # of the huddle except the user herself
         disp_recipients = [", ".join(recipient['email']
-                                for recipient in cast(Sequence[Mapping[str, Any]], get_display_recipient(mesg.recipient))
-                                    if recipient['email'] != user_profile.email)
+                                for recipient in cast(Sequence[Mapping[str, Any]],
+                                    get_display_recipient(mesg.recipient))
+                                        if recipient['email'] != user_profile.email)
```

Comment:
This probably should just be broken into multiple lines; it's basically unreadable.